src/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
     <head>
         <meta charset="utf-8">
-        <title>EveryBoard 25.1814-4.0</title>
+        <title>EveryBoard 25.1815-4.0</title>
```

Comment:
tu as bien updaté le code coverage and all ?
Oh bah tiens j'y pense, un check au moment du commit que le code-coverage est suffisamment récent, ça pourrait le faire comme ""update coverage" enforcer" non ?

Reply:
Ouip.

mouais  un check au commit ça veut dire que si t'es en train de travailler dur à quelque chose et que tu veux commiter du WIP, tu peux plus car t'as pas regénéré cela ? Pour l'instant ça s'assure qu'on découvre rien, est-ce non suffisant ? Si on couvre plus et qu'on oublie de générer, bah c'pas grave c'est en bonus pour le suivant !

Reply:
je voulais dire, est ce que le script qui fait tout les checks peut lui même marquer la branche comme caca si son coverage-timestamp est trop vieux, pas contourner "git commit" avec du code perso haha :D !

Reply:
ticketté après discussion, on aura le coverage qui fail à partir du moment où il n'est pas identique